src/main/kotlin/racingcar/RaceGame.kt

```diff
@@ -0,0 +1,21 @@
+package racingcar
+
+import racingcar.domain.Race
+import racingcar.view.InputView
+import racingcar.view.OutputView
+
+class RaceGame(private val inputView: InputView, private val resultView: OutputView) {
+
+    fun run() {
+        val carCount = inputView.carCount
+        val tryCount = inputView.tryCount
+        val race = Race(carCount)
+
+        println("실행 결과")
+
+        repeat((0 until tryCount).count()) {
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        repeat(tryCount) {
```

Reply:
앗! 그러네요 ㅋㅋ 감사합니다!